src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAXIMUM_POSSIBLE_BALL_NUMBER = 100;
```

Comment:
How do you think what max number will be returned by `random.nextInt(100);`?

Reply:
> How do you think what max number will be returned by `random.nextInt(100);`?

Thank you! Very observant advices!